src/gate_detect/launch/gate_detect_full.launch

```diff
@@ -8,17 +18,25 @@
             <param name="interpolationConstant1" value="81.88" type="double" />
             <param name="interpolationConstant2" value="-.92791" type="double" />
     </node>
-    <node name="hsv_filter_node" pkg="vision" type="hsv_filter_node" output="screen">
-        <param name="h_low" value="0" type="int" />
-        <param name="s_low" value="0" type="int" />
-        <param name="v_low" value="0" type="int" />
-        <param name="h_high" value="220" type="int" />
-        <param name="s_high" value="243" type="int" />
-        <param name="v_high" value= "200" type="int" />
-        <remap from="/camera/image_raw" to="/uwsim/camera2"/>
+
+    <node name="hsv_filter_node1" pkg="vision" type="hsv_filter_node" output="screen">
```

Comment:
It would be helpfule if we could make the `name` field into a descriptor of what these HSV filters are actually filtering for.  (ie. red, blue boxes, white lines, lines, :cow: :horse: , etc.)

Reply:
fixed